Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build dev test
+
+base:
+	docker build -t redis-py-base docker/base
+
+dev: base
+	docker-compose up -d --build
+
+test: dev
+	docker-compose run test find . -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete
+	docker-compose run test util/wait-for-it.sh master:6379 -- tox -e py3-plain -- --redis-url=redis://master:6379/9
+	docker-compose run test rm -rf .tox
```

Comment:
So... if you run `make test`, your `.tox` folder gets messed up (files owned by root). I'm not sure what to do about this, but it means that if you run `make test` and then later try to run `tox` locally (outside of a Docker container), the command will fail.

My solution? Nuke the `.tox` folder from the container-side after a test run.

Reply:
It might be easier to just COPY/ADD the redis-py project into the test container rather than using the VOLUME mount. The .pyc files can easily be prevented from getting copied with a .dockerignore file. It would also prevent someone from modifying a python file mid-test execution and having that being reflected via the VOLUME mount.

Reply:
That would help with test file cleanup for sure. Downside - we need to build the test image each time we run the command?

Reply:
That should be fast though assuming the COPY is at the end of the Dockerfile. All the previous commands will be cached.